Confidence/src/main/java/com/spotify/confidence/AndroidLifecycleEventProducer.kt

```diff
@@ -141,14 +136,12 @@ class AndroidLifecycleEventProducer(
 
         // Check and track Application Installed or Application Updated.
         if (previousBuild == null && legacyPreviousBuild == null) {
-            val message = mapOf("version" to currentVersion, "build" to currentBuild)
+            val message = mapOf(APP_VERSION_KEY to currentVersion, APP_BUILD_KEY to currentBuild)
             coroutineScope.launch { eventsFlow.emit(Event(APP_INSTALLED_EVENT, message)) }
         } else if (currentBuild != previousBuild) {
             val message = mapOf(
-                "version" to currentVersion,
-                "build" to currentBuild,
-                "previous_version" to (previousVersion ?: ConfidenceValue.String("")),
-                "previous_build" to (previousBuild ?: ConfidenceValue.String(""))
+                APP_VERSION_KEY to currentVersion,
+                APP_BUILD_KEY to currentBuild
             )
             coroutineScope.launch { eventsFlow.emit(Event(APP_UPDATED_EVENT, message)) }
```

Comment:
why do we sent these fields in the message part? aren't they already set in the context?

Reply:
Good point, maybe we shouldn't add them. Will change and align Swift